### PR TITLE
accept user input

### PR DESCRIPTION
**Pull request by bwfelix**
@pseudomuto

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
 def _main(name):
     print("Hello",name)
 
 
 if __name__ == '__main__':
     _main(NAME)
-    
\ No newline at end of file
+
+_main(sys.ARGV[1])
```
Comment (pseudomuto) [inline main.py]: 👮

File: main.py [hunk 2]
```diff
@@ -1,9 +1,10 @@
-NAME = "Ben"
+import sys
 
 def _main(name):
     print("Hello",name)
```
Comment (pseudomuto) [inline main.py]: /nit should have a space after the comma

**Review by pseudomuto — CHANGES_REQUESTED** [review]

**Comment by bwfelix** [comment]
Addressed style concerns, ready for re-review.

**Review by pseudomuto — APPROVED** [review]

**Pull request merged**